### ISSUE TITLE
nvda does not advertise videos in powerpoint

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9666]
### Steps to reproduce:
I created a PowerPoint presentation.
On the slide I inserted a video from the local machine.
I pressed f5 to start slideshow.
When reading the slide where there was the video the nvda read the slide with the arrows but did not report that there was a video on the slide.
I believe you have seen in previous calls this failure.
However the second part does not.
When pressing tab and the powerpoint should put the video in focus, however the nvda announces nothing.
Nvda should announce that the video has gained focus.
If I press enter after pressing tab and the video is played.
### Actual behavior:
nvda does not announce the existence of the video while walking with the arrows on the slide, and does not announce focus on the video by pressing tab.
### Expected behavior:
The nvda should announce that there is a video on the slide when walking with the arrows and should announce that the video gained focus by pressing tab.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
alfa 17460
#### Windows version:
10 17763
#### Name and version of other software in use when reproducing the issue:
16.0.4266.1001
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
log
[slide.txt](https://github.com/nvaccess/nvda/files/3248663/slide.txt)

**Comment by josephsl** [comment]
Hi,

Five years later...

@fernando-jose-silva, any updates?

Thanks.

**Comment by Adriani90** [comment]
Still reproducible with NVDA 2024.3 in Powerpoint 365. sofar all old and still open powerpoint issues are still reproducible according to my tests. Unfortunately Powerpoint is the piece of software that has  really  least  improved in terms of accessibility with NVDA during last 10 years or so.

**Comment by fernando-jose-silva** [comment]
Yes, the problem is reproducible, I tested it here too.
